main/res/values/preference_keys.xml

```diff
@@ -83,11 +83,8 @@
     <string translatable="false" name="pref_supersizeDistance">supersizeDistanceToggle</string>
     <string translatable="false" name="pref_plainLogs">plainLogs</string>
     <string translatable="false" name="pref_signature">signature</string>
-    <string translatable="false" name="pref_logTemplate1">pref_logTemplate1</string>
-    <string translatable="false" name="pref_logTemplate2">pref_logTemplate2</string>
-    <string translatable="false" name="pref_logTemplate3">pref_logTemplate3</string>
-    <string translatable="false" name="pref_logTemplate4">pref_logTemplate4</string>
-    <string translatable="false" name="pref_logTemplate5">pref_logTemplate5</string>
+    <string translatable="false" name="pref_fakekey_category_logTemplate">fakekey_category_logTemplate</string>
+    <string translatable="false" name="pref_logTemplates">pref_logTemplates</string>
```

Comment:
@moving-bits had suggested to remove the `pref_` in the key value... See a bit above https://github.com/cgeo/cgeo/pull/12300#discussion_r767251038

Reply:
Most preference use the pref_-prefix so I just reused it. Dunno what makes sense here